test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -1117,108 +1019,6 @@ test.describe( 'Image - interactivity', () => {
 			).not.toBeInViewport();
 		} );
 
-		test.describe( 'Animation Select visibility', () => {
```

Comment:
Removing this test as well, because there is no "animation selector" anymore. The Lightbox only has the "zoom" effect (animation) - it's the only one.